wot-wg-2023-details.html

```diff
@@ -274,13 +276,42 @@ <h2 id="geolocation-workitem">Geolocation</h2>
     </section>
     <section>
       <h2 id="onboarding-workitem">Onboarding</h2>
-      <p>In order to establish secure communications in some contexts, for example on a LAN, some form of onboarding
-      process to establish mutual trust between Things and Consumers is needed. Having a prescriptive (but optional)
-      process for this would be useful. Onboarding is part of the larger WoT Thing lifecycle which is described in the
-      Architecture document. In this work item we will define mechanisms to perform secure minimum-effort onboarding of
-      Things that conforms to the security and privacy requirements of specific deployment scenarios and use cases.
-      This work item complements the work that is done for device discovery, and is in particular useful when
-      registering TDs with a TD Directory service, however is also applicable for other use cases.</p>
+      <p>In order to establish secure communications in some contexts
+      a process to establish mutual trust between Things and Consumers is needed. 
+      Having a prescriptive (but optional)
+      process for this would be useful. 
+    </p><p>
+      In particular, the goal of the 
+      onboarding process would be allow encrypted communications to 
+      be used on a LAN, including establishing and exchanging
+      keying material among a set of devices without requiring fixed URLs
+      or use of a CA.
+      The keying material could, for example, take the form of certificates
+      containing public keys that can be used to identify devices and
+      also to establish mutual TLS or DTLS encrypted communication channels.
+      A pairwise onboarding process would involve the exchange of this
+      material between a Thing and a Consumer and a process or set of processes
+      that can be used to verify the level
+      of authorization to invoke affordances on the Thing.
+      In order to onboard sets of devices, group keys or a special "onboarding"
+      role can be used to establish transitive trust.
+    </p><p>
+      Onboarding is part of the larger WoT Thing lifecycle which is described in the
+      Architecture document. 
```

Comment:
**warning** I'm just noticing that if the architecture becomes not normative we will have a soft dependency with a non-normative document from a "normative but optional" text.

Reply:
Yes, I've been saying this.  If arch becomes non-normative, then certain normative content (current and planned) for security needs a new home - which means IMO a new normative security document.

Reply:
Yes, but the intention is that onboarding would be normative, too.  So it would be in the same category as the security assertions in the Architecture doc: it would need a new home if Architecture became informative.  One option would be to put it in Discovery, but it's not *really* discovery.  It could also go (along with security assertions) into a normative security doc.  HOWEVER, the point of this PR is just to define the work item, not to decide where it would live.